src/backend/effects/builtin/play-video.js

```diff
@@ -124,6 +124,38 @@ const playVideo = {
                 model="effect.twitchClipUsername"
                 placeholder-text="Ex: $streamer, $user, etc"
             />
+            <div
+                class="mt-6 mb-2"
+                ng-class="{'has-error': $ctrl.formFieldHasError('clipSeconds')}"
+            >
+                <div class="form-group flex-row jspacebetween" style="margin-bottom: 0;">
+                    <div>
+                        <label class="control-fb control--checkbox" style="margin:0;"> Maximum Clip Age
+                        <input type="checkbox" ng-model="effect.useMaxClipAge">
+                            <div class="control__indicator"></div>
+                        </label>
+                    </div>
```

Comment:
Please use the `<firebot-checkbox />` component for this bad boy. I know theres probably other checkboxes in this effect not using the "old way", I'm just trying to make sure at least that any new checkboxes we add are using the component :)